run.sh

```diff
@@ -58,4 +58,8 @@ docker run \
     -e GRAFANA_DATASOURCE_URL=$GRAFANA_DATASOURCE_URL \
     -e GRAFANA_DASHBOARD_URL=$GRAFANA_DASHBOARD_URL \
     -e USE_LOW_MEM_PRESSURE_STREAMING=$USE_LOW_MEM_PRESSURE_STREAMING \
+    -e KEYSTORE_PATH=KEYSTORE_PATH \
+    -e KEYSTORE_PASS=KEYSTORE_PASS \
+    -e KEY_PATH=KEY_PATH \
+    -e CERT_PATH=CERT_PATH \
```

Comment:
These new lines are setting the values to the literal names of the variables, ex. the variable `CERT_PATH` will have the value `"CERT_PATH"`. Is this intentional?

Reply:
No. This is a copy-paste error. I'll fix them in the next commit.